### PR TITLE
fix(migrate-v1): add learn more button to migrating v1 notification

### PR DESCRIPTION
**Pull request by kuojianlu**
Pending on #2132.

image

### CODE REVIEW

**Review by jayzhang — APPROVED** [review]

File: packages/vscode-extension/src/constants.ts [hunk 1]
```diff
@@ -3,3 +3,6 @@ export enum ConfigurationKey {
   ArmSupportEnabled = "armSupportEnabled",
   BicepEnvCheckerEnable = "validateBicep",
 }
+
+export const migrateV1DocUrl =
+  "https://github.com/OfficeDev/TeamsFx/blob/dev/docs/vscode-extension/migrate-v1/migrate-v1.md";
```
Comment (swatDong) [inline packages/vscode-extension/src/constants.ts:8]: Make this an aka link. (More flexible and we are able to get traffic report)

Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/constants.ts:8]: Do you use dev branch temporarily here?

Comment (kuojianlu) [reply packages/vscode-extension/src/constants.ts:8]: Will update to an aka link later.

Comment (kuojianlu) [reply packages/vscode-extension/src/constants.ts:8]: Good point, let me create an aka link for it.

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Review by swatDong — APPROVED** [review]

**Pull request merged**